KWVerificationCodeView/Classes/KWVerificationCodeView.swift

```diff
@@ -156,10 +148,6 @@ public protocol KWVerificationCodeViewDelegate: class {
   }
 
   // MARK: - Public Methods
-  public func focus() {
```

Comment:
Any particular reason you removed the focus() method? It was added as some of the users requested for it.